src/register.js

```diff
@@ -37,11 +59,15 @@ export default function registerStories(req, fileName, sbInstance, plugins) {
         }
       };
     };
-
+    decorators.forEach((decor) => {
```

Comment:
If you omit `decorators` as an argument to `registerStories`, won't this blow up since `decorators` would be undefined? I think we should guard this logic by first checking whether `decorators` is present.

Reply:
Good catch, yeah it definitely would break. I've updated it